packages/input-stepper/src/LionInputStepper.js

```diff
@@ -195,61 +235,67 @@ export class LionInputStepper extends LionInput {
   }
 
   /**
-   * Override after template to none
+   * Override before template to none
    * @override
    */
   // eslint-disable-next-line class-methods-use-this
-  _inputGroupAfterTemplate() {
+  _inputGroupBeforeTemplate() {
```

Comment:
I didn't see review first version of the stepper, but I was wondering why do we override these?

Reply:
I think we need it otherwise `before` slot will show the text/content above the text box which is not expected.

Reply:
1. It only appears if you add anything to the slot
2. You can style the way you want it (most likely inline) since this is a white label component. 
3. We allow it for all other input fields, so i don't see a reason to disallow it for the stepper.

Reply:
Okay, let's allow it. 😅